rdf-star-cg-spec.html

```diff
@@ -106,15 +106,69 @@ <h2>Background and Motivation</h2>
     <section class="informative">
       <h2>Overview</h2>
 
-      <p>TODO (the purpose of this section will be to provide an informal introduction to the approach for practitioners)</p>
+<p>The RDF data model lets you state facts in three-part subject-predicate-object statements known as triples. For example, with a single RDF triple you can say that employee38 has a hireDate of "2020-04-03". A triple's predicate is a property specified with an IRI (an Internationalized version of a URI) to identify the namespace of the property name. A triple's subject and object can each be an IRI referencing any entity, and the object can also be a literal value such as the date 2020-04-03 or data of other types such as strings, numbers, or Boolean values. </p>
+
+<p>The subject and object of a triple can themselves reference triples. In the statement "employee22 claims that employee38 has a familyName of 'Smith'", the object of the triple that has employee22 as its subject is the statement "employee38 has a familyName of 'Smith'". This use of a triple as the subject or object resource of another triple so that we can say things about that triple is known as <a href='https://www.w3.org/TR/1999/REC-rdf-syntax-19990222/#higherorder'>reification</a>. </p>
```

Comment:
```suggestion
<p>The subject and object of a triple can themselves reference triples. In the statement "employee22 claims that employee38 has a jobTitle of 'Assistant Designer'", the object of the triple that has employee22 as its subject references the statement "employee38 has a jobTitle of 'Assistant Designer'". This use of a triple as the subject or object resource of another triple so that we can say things about that triple is known as <a href='https://www.w3.org/TR/1999/REC-rdf-syntax-19990222/#higherorder'>reification</a>. </p>
```